packages/sanity/src/structure/documentActions/PublishAction.tsx

```diff
@@ -176,6 +176,7 @@ export const PublishAction: DocumentActionComponent = (props) => {
 
   const disabled = Boolean(
     publishScheduled ||
+      editState.ready ||
```

Comment:
I might be mistaken, but shouldn't this be `!editState.ready`?

Reply:
Hm, of course you're right... But I actually want to reset this commit. I had a conversation with Bjørge about it and he pointed out using the `editState.ready` - but this does not take in to consideration when there are validation issues, which is where this delay is most apparent. I can add this to the check as well though.

Reply:
@ninaandal I'm not _super_ familiar with this part of the codebase yet, but looking at `packages/sanity/src/core/store/_legacy/document/document-pair/editState.ts`, I think you're correct that `editState.ready` doesn't reflect the validation state. It looks like `editState.ready` simply reflects whether the document pair is available (it's only `false` until a value is emitted from `snapshotPair`).